install/helm/agones/templates/controller.yaml

```diff
@@ -36,7 +37,11 @@ spec:
   template:
     metadata:
       annotations:
+{{- if $featureGates.SplitControllerAndExtensions }}
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

Comment:
Is there a difference between explicitly setting this to true vs. just leaving it off of the deployment entirely? Is it better to be explicit about it? I noticed in the extensions deployment we left it off.

Reply:
Let's omit it. Technically `true` is [broader in scope than the omission](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node) but I don't think any of the conditions applies to us. (It's not a `kube-system` pod, it uses ephemeral not local storage, and the controller is backed by a deployment.)